eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -84,6 +84,8 @@ case class Commitments(channelVersion: ChannelVersion,
 
   def hasNoPendingHtlcs: Boolean = localCommit.spec.htlcs.isEmpty && remoteCommit.spec.htlcs.isEmpty && remoteNextCommitInfo.isRight
 
+  def hasNoPendingFeeUpdate: Boolean = (localChanges.all ++ remoteChanges.all).collectFirst { case _: UpdateFee => true }.isEmpty && remoteNextCommitInfo.isRight
```

Comment:
I believe if want this method to be consisten with `hasNoPendingHtlcs` (and with the comments in Channel.scala), it should be:

```suggestion
  def hasNoPendingFeeUpdate: Boolean = (localChanges.signed ++ localChanges.acked ++ remoteChanges.acked ++ remoteChanges.signed).collectFirst { case _: UpdateFee => true }.isEmpty && remoteNextCommitInfo.isRight
```

Reply:
You're right! Done in https://github.com/ACINQ/eclair/pull/1661/commits/200c2c03f98348bb8387359bdad91641f0d0998d